RobotCode/snobot2018/src/org/snobot/leds/SnobotLedManager.java

```diff
@@ -22,7 +27,12 @@
 public class SnobotLedManager implements ILedManager
 {
     private static final int NUM_LEDS = 144;
-    private static final int NUM_LEDS_TOP_PORTION = 14;
+    private static final int NUM_LEDS_TOP_PORTION = 3;
+
+    private static final Color RAW_RED = Color.red;
+    private static final Color RAW_GREEN = Color.green;
+    private static final Color RAW_BLUE = Color.blue;
+    private static final Color RAW_ORANGE = Color.orange;
 
     private static final Color SNOBOT_BLUE = Color.blue;
```

Comment:
The raw colors should be left alone, but we might change these to match our "branding" scheme (if we have one).  i.e. make the blue a navy blue to match our shirt

Reply:
where should we do that